modules/project-docs/pages/sdk-release-notes.adoc

```diff
@@ -28,6 +28,31 @@ Older versions may work, but are not supported.
 
 The most current and up to date API Documentation is always available through the https://pkg.go.dev/github.com/couchbase/gocb/v2[godoc website].
 
+== Version 2.4.0 (11 February 2021)
```

Comment:
Guessing this date will change as it's not available yet?

Reply:
Correct.